srcpkgs/spek-X/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'spek-X'
+pkgname=spek-X
+version=0.9.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel intltool libtool pkg-config wxWidgets-common"
+makedepends="ffmpeg-devel wxWidgets-gtk3-devel"
+depends="ffmpeg"
+short_desc="Acoustic spectrum analyser (fork of spek-alternative)"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/MikeWang000000/spek-X"
+distfiles="https://github.com/MikeWang000000/spek-X/archive/v${version}.tar.gz"
+checksum=d6cf8dac3b0a3749c0128d204393a633c28f863d9886fbec97e5d5924dad53ef
+conflicts="spek-alternative"
+
+# Tell autogen.sh to use "wx-config-gtk3" instead of "wx-config"
+export WX_CONFIG_NAME="wx-config-gtk3"
```

Comment:
could this be done with `env WX_CONFIG_NAME=... NOCONFIGURE=1 ./autogen.sh`?

Reply:
Doesn't work for me.